hal/architecture/AVR/MyHwAVR.h

```diff
@@ -28,7 +28,12 @@
 #include <avr/wdt.h>
 #include <avr/boot.h>
 #include <util/atomic.h>
-#include <SPI.h>
+
+#if defined( __AVR_ATtiny25__ ) || defined( __AVR_ATtiny45__ ) || defined( __AVR_ATtiny85__ ) || defined( __AVR_ATtiny87__ ) || defined( __AVR_ATtiny167__ )	
+    #include <TinySoftwareSPI.h>
```

Comment:
What library is this? Is it bart of the standard Arduino library or must it be i stalled manually?

Reply:
This library comes along with most (can't guarantee all) Tiny implementations that I've tried. Makes MySensors work for my Digispark Pro (ATTinty167) like a charm.

Reply:
TinySoftwareSPI.h was removed from AttinyCore. And I don't know the reason